src/components/TokenGeneralInfo.js

```diff
@@ -58,20 +59,20 @@ class TokenGeneralInfo extends React.Component {
   /**
    * Upadte token info getting data from the full node (can mint, can melt, total supply)
```

Comment:
Should we update this description?

Reply:
We should, done in d012a01736dfdfea09cb341cce722ef943049adc